src/parser.cpp

```diff
@@ -6091,7 +6097,7 @@ gb_internal String build_tag_get_token(String s, String *out) {
 }
 
 gb_internal bool parse_build_tag(Token token_for_pos, String s) {
-	String const prefix = str_lit("+build");
+	String const prefix = str_lit("build");
```

Comment:
+ is now removed earlier because we do a deprecation check outside of `parse_file_tag` that also skips over the +